llvm/test/Transforms/LoopVectorize/predicate-switch.ll

```diff
@@ -162,21 +343,21 @@ define void @switch_to_header(ptr %start) {
 ; IC1-NEXT:  [[ENTRY:.*]]:
 ; IC1-NEXT:    br label %[[LOOP_HEADER:.*]]
 ; IC1:       [[LOOP_HEADER]]:
-; IC1-NEXT:    [[IV:%.*]] = phi i64 [ 0, %[[ENTRY]] ], [ [[IV_NEXT:%.*]], %[[LOOP_HEADER_BACKEDGE:.*]] ]
+; IC1-NEXT:    [[IV:%.*]] = phi i64 [ 0, %[[ENTRY]] ], [ [[IV_NEXT:%.*]], %[[IF_THEN1:.*]] ]
```

Comment:
If/how are these changes in block labels (and those below) related to this patch?

Reply:
Re-generated check lines separately, changes here should be gone now 5286656609418c766272ff78e62fda41671ef4cb